content/source/docs/enterprise/api/ptfe_settings.html.md

```diff
@@ -0,0 +1,491 @@
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/admin/settings/smtp
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "smtp-settings",
+    "type": "settings",
+    "attributes": {
+      "enabled": true,
+      "host": "example.com",
+      "port": 25,
+      "sender": "sample_user@example.com",
+      "auth": "login",
+      "username": "sample_user"
+    }
+  }
+}
+```
+
+## Update SMTP Settings
+`PATCH /api/v2/admin/settings/smtp`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
+
+Status  | Response                                     | Reason
+--------|----------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "settings"`) | Successfully updated the SMTP settings
+[401][] | [JSON API error object][]                    | Failure during test message delivery, SMTP user credentials are invalid
+[404][] | [JSON API error object][]                    | User unauthorized to perform action
+[422][] | [JSON API error object][]                    | Malformed request body (missing attributes, wrong types, etc.)
+[500][] | [JSON API error object][]                    | Failure during test message delivery, SMTP server server error
+[504][] | [JSON API error object][]                    | Failure during test message delivery, SMTP server timeout
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required if `data.attributes.enabled` is set to `true`.
+
+Key path                    | Type   | Default | Description
+----------------------------|--------|---------|------------
+`data.attributes.enabled`   | bool   | `false` | Allows SMTP to be used. If true, all remaining attributes are required.
```

Comment:
…except username & password if auth is `"none"`, I guess?

Reply:
True, which I call out in the username and password descriptions. Should I be extra explicit here as well?

Reply:
No, I think it's clear enough as-is!